### PR TITLE
Update load_file op. to add load_options param in document

### PR DESCRIPTION
**Pull request by utkarsharma2**
# Description
## What is the current behavior?
Update document to add load_options parameter in load_file() operator. 

closes: #1559 #1558 

## What is the new behavior?
1. Updated example dag to showcase the use of the `load_options` parameter
2. Updated document to reflect the new `load_options` parameter.

## Does this introduce a breaking change?
Nope

### CODE REVIEW

File: python-sdk/docs/astro/sql/operators/load_file.rst [hunk 1]
```diff
@@ -76,6 +76,8 @@ Parameters to use when loading a file to a database table
        :start-after: [START load_file_example_3]
        :end-before: [END load_file_example_3]
 
+#. **load_options** - `load_options`_
```
Comment (tatiana) [inline python-sdk/docs/astro/sql/operators/load_file.rst:79]: We probably want to give more details about what `load_options` is an which values it takes, referencing examples where possible

File: python-sdk/docs/astro/sql/operators/load_file.rst [hunk 2]
```diff
@@ -116,6 +118,7 @@ Parameters to use when loading a file to a Pandas dataframe
        :start-after: [START load_file_example_6]
        :end-before: [END load_file_example_6]
 
+#. **load_options** - `load_options`_
```
Comment (tatiana) [inline python-sdk/docs/astro/sql/operators/load_file.rst:121]: Same here; the description is not helping to understand what these options are. It is probably worth to also say that the `native_support_kwargs` will be replaced by this

File: python-sdk/docs/astro/sql/operators/load_file.rst (hunk 1 above)
Comment (utkarsharma2) [reply python-sdk/docs/astro/sql/operators/load_file.rst:79]: @tatiana Just checking, I have added this in one section at the end of this file and referenced it here. Below is the content of that section, do you feel we need more here?
```
We can use this parameter to pass options that can be used by integrations SDK use internally. For example, there can be a simple case of passing a ``delimiter`` while loading CSV to pandas.read_csv() method.

    .. literalinclude:: ../../../../example_dags/example_load_file.py
       :language: python
       :start-after: [START load_file_example_22]
       :end-before: [END load_file_example_22]

    ``load_options`` is a list of different :py:class:`LoadOptions <astro.options.LoadOptions>` class objects. Please refer the list below for all valid options.

    - :py:class:`PandasCsvLoadOptions <astro.dataframes.load_options.PandasCsvLoadOptions>`
    - :py:class:`PandasJsonLoadOptions <astro.dataframes.load_options.PandasJsonLoadOptions>`
    - :py:class:`PandasNdjsonLoadOptions <astro.dataframes.load_options.PandasNdjsonLoadOptions>`
    - :py:class:`PandasParquetLoadOptions <astro.dataframes.load_options.PandasParquetLoadOptions>`
    - :py:class:`DeltaLoadOptions <astro.databases.databricks.load_options.DeltaLoadOptions>`
    - :py:class:`SnowflakeLoadOptions <astro.options.SnowflakeLoadOptions>`
```

File: python-sdk/tests_integration/test_example_dags.py [hunk 3]
```diff
@@ -88,8 +88,9 @@ def order(dag_id: str) -> int:
 
 @pytest.mark.parametrize("dag_id", sorted(dag_bag.dag_ids, key=order))
 def test_example_dag(session, dag_id: str):
-    dag = dag_bag.get_dag(dag_id)
-    wrapper_run_dag(dag)
+    if dag_id == "example_load_file":
+        dag = dag_bag.get_dag(dag_id)
+        wrapper_run_dag(dag)
```
Comment (feluelle) [inline python-sdk/tests_integration/test_example_dags.py:93]: I assume this is only temporarily for local testing. But actually you could have just used `test_example_dag[example_load_file]`.

**Comment by utkarsharma2** [comment]
Closing in favour of - https://github.com/astronomer/astro-sdk/pull/1567

**Pull request closed without merging**